semgrep_metrics.atd

```diff
@@ -66,6 +66,8 @@ type payload = {
 type environment = {
     (* semgrep version *)
     version <ocaml mutable>: string;
+    os: string;
```

Comment:
How will the backend react to those new fields? I guess we're not using yet semgrep_metrics.atd on the backend side
so I guess it's fine, but better to think about it.

Maybe you can also add a comment before like (* since Semgrep 1.59 *).

also isJS is a bit vague, so would be better to explain what it is, and for os maybe give an example (* e.g., "linux" *)

Reply:
From what I've seen the backend usually manually accesses fields i.e. data["version"] so new fields should be ok, I couldn't find a breaking instance, but there may be a place I missed where this data is reported

Reply:
I'll add comments, not sure what to call isJS, maybe isTranspiledJS or isJSOO?